### ISSUE TITLE
Food manager wordpress.org page in details content are related to event.

### ISSUE THREAD

**Issue by patel-nimesh** [wpfoodmanager/wp-food-manager#343]
On the Food manager core plugin wordpress.org page content are not proper. They are related to event

https://wordpress.org/plugins/wp-food-manager/

![image](https://user-images.githubusercontent.com/121149500/223019034-c463dfde-f988-4ef1-8897-0c8764cbd217.png)
![image](https://user-images.githubusercontent.com/121149500/223019083-6fe39e9d-ab1b-4321-815f-8956c03e59ce.png)
![image](https://user-images.githubusercontent.com/121149500/223019163-684a8f9d-f6a7-4729-877f-a431d81f918a.png)
![image](https://user-images.githubusercontent.com/121149500/223019199-0ada0189-3f8b-4414-9750-86ea3c678c3e.png)

**Comment by ashokdudhat** [comment]
@krinaydhanani 

Hi Ashok,

Thanks for your email. The work we provided for you is good: you are correct that some of the copy uses the original text; this is because we felt the original text made good points! Changing the text now is just going to make it different, not better. I'd be happy to make a correction if I felt we'd done a poor job, but we haven't – the copy here is good, and we reviewed it on a call at the time and we all agreed it was good. I'm sorry I can't be more helpful here: I do stand by the work we did on this, and wish you all the best.


Best,

Alex.
